src/EdFi.AnalyticsMiddleTier.Tests/EdFi.AnalyticsMiddleTier.Tests.csproj

```diff
@@ -1,5 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+  
```

Comment:
Team should carefully evaluate before accepting a change in language. At the moment I'm not expressing an opinion, as I haven't given it any thought yet. Just want to dream the team's attention to this.

Reply:
On further review this looks appropriate. We were implicitly using language version 8.0 with VS2019. This just makes it more apparent and usable in other tools.